code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -57,47 +59,64 @@
 		else
 			src << "You moved out of range, and were pulled back! You can only move [range] meters from [summoner.real_name]"
```

Comment:
This is missing the `<span class='holoparasite'>` that other messages of this kind have